plot_data/core.py

```diff
@@ -1212,13 +1209,13 @@ def get_csv_vectors(filepath):
                  'primitivegroup': PrimitiveGroup, "scattermatrix": ScatterMatrix}
 
 
-def bounding_box(plot_datas: Subclass[PlotDataObject]):
+def bounding_box(plot_datas: List[PlotDataObject]):
     """
     Calls input plot_datas' bounding_box method, if it has one.
 
     :param plot_datas: The target object the bounding_box method has to\
      be called from
-    :type plot_datas: Subclass[PlotDataObject]
+    :type plot_datas: PlotDataObject
```

Comment:
The typing went from `Subclass[PlotDataObject]` to `List[PlotDataObject]`. According to your other changes, typing only with `PlotDataObject` should be correct.
Moreover, in the docstring, the typing went from `Subclass[PlotDataObject]` to `PlotDataObject`.

Reply:
The use of the plot_datas variable in the method lead me to think that it should be a list, because we loop on it. Moreover, if I understand it well, this takes several plot_datas and find the bounding box of the Union of it. So I think that the previous typing was wrong, but it might not be the case here